lib/themes/reps/template.php

```diff
@@ -421,21 +421,36 @@ function reps_process_entity(&$variables) {
       'reps_core_sb_right_image_link',
       'reps_core_sb_right_blue_button',
     ))) {
-    $variables['content']['#markup']['title'] = l(
-      $variables['content']['title_field']['#items'][0]['value'],
-      $variables['content']['field_reps_core_external_url']['#items'][0]['original_url'],
-      array('attributes' => $variables['content']['field_reps_core_external_url']['#items'][0]['attributes'])
-    );
-
-    if ($variables['bean']->type = 'reps_core_sb_right_image_link') {
-      $variables['content']['#markup']['image'] = l(
-        render($variables['content']['field_reps_core_image']),
+    if (!empty($variables['content']['title_field'])) {
+      $variables['content']['#markup']['title'] = l(
+        $variables['content']['title_field']['#items'][0]['value'],
         $variables['content']['field_reps_core_external_url']['#items'][0]['original_url'],
-        array(
-          'attributes' => $variables['content']['field_reps_core_external_url']['#items'][0]['attributes'],
-          'html' => TRUE,
-        )
+        array('attributes' => $variables['content']['field_reps_core_external_url']['#items'][0]['attributes'])
       );
+      if ($variables['bean']->type = 'reps_core_sb_right_image_link') {
+        $variables['content']['#markup']['image'] = l(
+          render($variables['content']['field_reps_core_image']),
```

Comment:
Hmm this can produce HTML. Do not use render, go fetch the value of the field with [0][LANGUAGE_NONE]['value']

Reply:
I've moved to this solution after Julie feedback during the first implementation: https://github.com/digit-devs/reps-france-dev/pull/2#discussion_r517191494

Reply:
Ok